recipes/systemc/meta.yaml

```diff
@@ -0,0 +1,117 @@
+{% set name = "systemc" %}
+{% set version = "2.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/accellera-official/systemc/archive/refs/tags/{{ version }}.tar.gz
+  sha256: bfb309485a8ad35a08ee78827d1647a451ec5455767b25136e74522a6f41e0ea
+  patches:
+    - patches/0001-examples-sysc-2.3-simple_async-needs-PTHREADS.patch  # https://github.com/accellera-official/systemc/pull/44
```

Comment:
This patch needs to be linux-only

Reply:
Use a selector?

Reply:
Did this in [4b65a1c](https://github.com/conda-forge/staged-recipes/pull/22280/commits/4b65a1cfad89ff47dc9544fff4008167d3dbeda2)